app/src/org/commcare/activities/SyncUIHandling.java

```diff
@@ -0,0 +1,85 @@
+package org.commcare.activities;
+
+import org.commcare.interfaces.ConnectorWithResultCallback;
+import org.commcare.logging.analytics.GoogleAnalyticsFields;
+import org.commcare.logging.analytics.GoogleAnalyticsUtils;
+import org.commcare.tasks.DataPullTask;
+import org.commcare.tasks.ResultAndError;
+import org.javarosa.core.services.locale.Localization;
+
+/**
+ * Default sync task result handling logic.
+ *
+ * @author Phillip Mates (pmates@dimagi.com)
+ */
+public class SyncUIHandling {
```

Comment:
OOC do you think this class and the ConnectorWithResultCallback will be used outside of the home activity?